lib/fluent-plugin-s3-arrow/schemas/aws_glue.rb

```diff
@@ -0,0 +1,113 @@
+require 'aws-sdk-glue'
+
+module FluentPluginS3Arrow
+  module Schemas
+    class AWSGlue
+      class Error < RuntimeError; end
+      class ConvertError < Error; end
+      class Field < Struct.new(:name, :type); end
+
+      def initialize(table_name, options={})
```

Comment:
Using the keyword arguments feature instead of pseudo keyword arguments by `Hash` is better:

```suggestion
      def initialize(table_name, **options)
```